### PR TITLE
Adding a couple PHP security rules

### PR DESCRIPTION
**Pull request by rezen**
- Added a rule to check for loose equality checks involving md5
- Added a rule to check for CURLOPT_SSL_VERIFYPEER set to false

### CODE REVIEW

**Comment by ievans** [comment]
thanks for these rules @rezen! I got your CLA, @cla-bot check

File: php/lang/security/md5-loose-equality.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  patterns:
+  - pattern-either:
+    - pattern: $X == $FUNC(...)
+    - pattern: $FUNC(...) == $X
+    - pattern: $FUNC(...) == $FUNC(...)
+  - metavariable-regex:
+      metavariable: $FUNC
+      regex: md5|md5_file
+  message: |
+    Make sure comparisons involving md5 values are strict to avoid type juggling issues
```
Comment (ievans) [inline php/lang/security/md5-loose-equality.yaml]: Does "strict" mean using the `===` operator? If so, perhaps the message could be made more actionable

**Review by ievans — APPROVED** [review]
Looks great to me!

File: php/lang/security/md5-loose-equality.yaml (hunk 1 above)
Comment (rezen) [reply php/lang/security/md5-loose-equality.yaml]: It does 🙂 would you like me to update the message?

**Review by minusworld — APPROVED** [review]

**Pull request merged**